lib/api/reviews/client.ts

```
@@ -124,22 +124,75 @@ export function useReviewClient() {
+        id: apiReview.id,
+        providerId: apiReview.provider_id,
+        userId: apiReview.user_id,
+        rating: apiReview.rating,
+        comment: apiReview.comment,
+        status: apiReview.status,
+        createdAt: apiReview.created_at,
+        updatedAt: apiReview.updated_at
+      };
+      
+      return transformedReview;
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)